content/en/real_user_monitoring/feature_flag_tracking/setup.md

```diff
@@ -0,0 +1,849 @@
+---
+title: Setup Feature Flag Tracking
+beta: true
+description: Learn how to set up RUM to capture feature flag data and analyze the performance in Datadog
+aliases:
+- /real_user_monitoring/guide/getting-started-feature-flags/
+- /real_user_monitoring/guide/setup-feature-flag-data-collection/
+disable_toc: false
+further_reading:
+- link: "/real_user_monitoring/guide/setup-feature-flag-data-collection/"
+  tag: "Documentation"
+  text: "Set up Feature Flag data collection"
+- link: "/real_user_monitoring/explorer/"
+  tag: "Documentation"
+  text: "Learn about the RUM Explorer"
+- link: "https://www.datadoghq.com/blog/feature-flag-tracking/"
+  tag: "Blog"
+  text: "Ensure release safety with feature flag tracking in Datadog RUM"
+---
+
+Feature flag data provides greater visibility into user experience and performance monitoring. It allows you to determine which users are being shown a specific feature and assess if any changes introduced are impacting user experience or negatively affecting performance.
+
+By enriching your RUM data with feature flag data, you can be confident that your feature successfully launches without unintentionally causing a bug or performance regression. With this additional layer of insight, you can correlate feature releases with performance, pinpoint issues to specific releases, and troubleshoot faster.
+
+## Set up RUM monitoring
+
+Feature flag tracking is available in the RUM Browser, iOS, Android, Flutter, and React Native SDK.
+
+{{< tabs >}}
+{{% tab "Browser" %}}
+
+To enable feature flag data collection for the Browser SDK:
+
+1. Set up [RUM browser monitoring][1]. You need the Browser RUM SDK version >= 4.25.0.
+
+2. Initialize the RUM SDK and configure the `enableExperimentalFeatures` initialization parameter with ` ["feature_flags"]`.
+
+   <details open>
+     <summary>npm</summary>
+
+   ```javascript
+     import { datadogRum } from '@datadog/browser-rum';
+
+     // Initialize Datadog Browser SDK
+     datadogRum.init({
+       ...
+       enableExperimentalFeatures: ["feature_flags"],
```

Comment:
**💬 suggestion:** ‏since v5.17.0, you don't need `enableExperimentalFeatures: ["feature_flags"],` anymore. Remove this?